### PR TITLE
{bio}[foss/2022b] KrakenUniq-1.0.3-GCC-11.3.0.eb

### PR DESCRIPTION
**Pull request by deniskristak**
https://github.com/vscentrum/vsc-software-stack/issues/102

### CODE REVIEW

**Comment by deniskristak** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/k/KrakenUniq/KrakenUniq-1.0.3-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+description = """KrakenUniq: confident and fast metagenomics classification using unique k-mer counts"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+github_account = 'fbreitwieser'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['559b45081688118b4872e1b9153f83d0ae3414649a1b299ab4f9996d9e154403']
+
+dependencies = [
```
Comment (casparvl) [inline easybuild/easyconfigs/k/KrakenUniq/KrakenUniq-1.0.3-foss-2022a.eb:16]: Seeing the error on Generoso, I think it needs Perl as a dependency (it seems to be missing a particular Perl package when running the `sanity_check_commands`)

File: easybuild/easyconfigs/k/KrakenUniq/KrakenUniq-1.0.3-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+easyblock = 'Binary'
+
+name = 'KrakenUniq'
+version = '1.0.3'
+
+homepage = 'https://github.com/fbreitwieser/krakenuniq/'
+description = """KrakenUniq: confident and fast metagenomics classification using unique k-mer counts"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
```
Comment (casparvl) [inline easybuild/easyconfigs/k/KrakenUniq/KrakenUniq-1.0.3-foss-2022a.eb:9]: Does `KrakenUniq` require `foss` as a toolchain? Isn't GCC enough here?

**Review by casparvl — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/k/KrakenUniq/KrakenUniq-1.0.3-foss-2022a.eb (hunk 2 above)
Comment (deniskristak) [reply easybuild/easyconfigs/k/KrakenUniq/KrakenUniq-1.0.3-foss-2022a.eb]: done

**Comment by deniskristak** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/5c2e78c45f6e18c1b4e12ddb47ab7927 for a full test report.

**Comment by verdurin** [comment]
```
Compress::Raw::Zlib version 2.103 required--this is only version 2.101 at /eb/maint/software/Perl/5.34.1-GCCcore-11.3.0/lib/perl5/site_perl/5.34.1/IO/Uncompress/RawInflate.pm line 8.
```

**Comment by smoors** [comment]
@deniskristak i've added the `Compress-Raw-Zlib` dep to get around the `Compress::Raw::Zlib` version error

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @deniskristak!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3102.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/2644f930a9b048aeccb2753f72caeb0a for a full test report.

**Pull request merged**